.gitignore

```diff
@@ -15,6 +15,7 @@ pids
 libs/
 node_modules/
 bower_components/
+*.js.map
```

Comment:
Isn't that configurable? I mean to put the directory for maps into /tmp directory or something like that.